ros_ws/src/extractor_context.txt

```diff
@@ -0,0 +1,7 @@
+You are an expert in grammatical structures and will be given a sentence from which 
+you should extract what exactly is being requested. It will likely be the direct object in a 
```

Comment:
Don't insert new lines if you don't need to, especially in the middle of contextual thoughts. The model doesn't care but it makes it harder for formatting/parsing down range.

Reply:
I pulled them out in the read line.  I thought it made it easier to read here.